ember_debug/vendor/startup-wrapper.js

```diff
@@ -277,4 +288,74 @@ var EMBER_VERSIONS_SUPPORTED = {{EMBER_VERSIONS_SUPPORTED}};
     return !toVersion || compareVersion(version, toVersion) === -1;
   }
 
+  function onReady(callback) {
+    if (document.readyState === 'complete' || document.readyState === 'interactive') {
+      setTimeout(completed);
+    } else {
+      document.addEventListener("DOMContentLoaded", completed, false);
+      // For some reason DOMContentLoaded doesn't always work
+      window.addEventListener("load", completed, false);
+    }
+
+    function completed() {
+      document.removeEventListener("DOMContentLoaded", completed, false);
+      window.removeEventListener("load", completed, false);
+      callback();
+    }
+  }
+
+  /**
+   * Compares two Ember versions.
+   *
+   * Returns:
+   * `-1` if version1 < version
+   * 0 if version1 == version2
+   * 1 if version1 > version2
+   *
+   * @param {String} version1
+   * @param {String} version2
+   * @return {Boolean} result of the comparison
+   */
+  function compareVersion(version1, version2) {
```

Comment:
Could we import `compareVersion` and `cleanupVersion` from here? https://github.com/emberjs/ember-inspector/blob/master/ember_debug/utils/version.js

Reply:
@rwwagner90 We can't import those functions because this code isn't allowed to import anything during the period where it's checking the version.

I explained it in more detail in the commit message, but the short version is that if we require anything here, some pieces of the current ember_debug.js will stick around even after we load the earlier version of ember_debug.js, creating a big frankenstein mess. That was the root cause of the original bug.